### DISCUSSION TITLE
v2中，怎样处理Field的onChange、onBlur、onClick等事件？

### DISCUSSION THREAD

**Discussion by effapp** [alibaba/formily#1272] [Q&A 2.x]
v2中，怎样处理Field的onChange、onBlur、onClick等事件？能写个简单的代码片段最好。

**Comment by janryWang** [comment]
没懂你的意思，描述清楚点，你到底是想干啥

**Comment by effapp** [comment]
我想在Field的reactions里处理字段值改变后，向服务端发送值

**Reply by janryWang** [reply]
https://v2.formilyjs.org/zh-CN/guide/advanced/validate#%E8%81%94%E5%8A%A8%E6%A0%A1%E9%AA%8C

**Comment by effapp** [comment]
能不能再field的reactions中判断当前是onChange、onBlur或者别的什么事件吗？

**Reply by janryWang** [reply]
不能，reactions是响应的数据变化，不是事件触发，如果你非要监听事件，给组件手动传onChange/onBlur事件就行
